serverless.yml

```diff
@@ -1,6 +1,8 @@
 service: airtable-backups-demo
 
-frameworkVersion: "=1.46.0"
+frameworkVersion: "^2.30.2"
+
+configValidationMode: error
```

Comment:
Is that a new option?

Reply:
Yes, warnings will become errors by default in serverless 3.0 and this option turns that on now.